src/__test__/persistentSubscription/connectToPersistentSubscriptionToAll.test.ts

```diff
@@ -0,0 +1,737 @@
+/** @jest-environment ./src/__test__/utils/enableVersionCheck.ts */
+
+import { pipeline, Writable, Readable, Transform } from "stream";
+import { promisify } from "util";
+
+import {
+  createTestCluster,
+  Defer,
+  delay,
+  jsonTestEvents,
+  matchServerVersion,
+  optionalDescribe,
+  postEventViaHttpApi,
+} from "../utils";
+
+import {
+  AllStreamResolvedEvent,
+  NotLeaderError,
+  PersistentSubscription,
+  EventStoreDBClient,
+  jsonEvent,
+  persistentSubscriptionToAllSettingsFromDefaults,
+  START,
+  UnsupportedError,
+  streamNameFilter,
+  END,
+} from "../..";
+
+const asyncPipeline = promisify(pipeline);
+
+describe("connectToPersistentSubscriptionToAll", () => {
```

Comment:
**SUGGESTION:** That might be out of the scope of this PR, but it'd be good to add more tests around handling node loss, reconnecting etc. It's not a blocking suggestion, but as there are always many support issues for that feature, it might be worth enhancing those edge cases coverage.

Reply:
Agreed, I've opened an issue here: #231